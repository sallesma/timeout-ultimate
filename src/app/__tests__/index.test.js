import React from 'react';
import { render } from '@testing-library/react-native';

import IndexScreen from '../index';

describe('<IndexScreen />', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<IndexScreen />);

    expect(toJSON()).toMatchSnapshot();
  });
});
