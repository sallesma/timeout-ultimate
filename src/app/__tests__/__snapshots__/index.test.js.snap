// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders correctly 1`] = `
<View
  style={
    {
      "flex": 1,
      "justifyContent": "space-evenly",
      "paddingHorizontal": 16,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "height": 100,
        "marginVertical": 8,
      }
    }
  >
    <Image
      source={1}
      style={
        {
          "borderRadius": 50,
          "flex": 1,
          "height": 100,
          "width": 100,
        }
      }
    />
  </View>
  <Text
    style={
      {
        "color": "#2a9d9d",
        "fontSize": 32,
        "fontWeight": "bold",
        "margin": 16,
        "textAlign": "center",
      }
    }
  >
    Timeout Ultimate
  </Text>
  <Text
    style={
      {
        "fontSize": 20,
        "textAlign": "center",
      }
    }
  >
    <Text
      style={
        {
          "color": "#2a9d9d",
          "fontSize": 26,
          "fontWeight": "bold",
        }
      }
    >
      268
    </Text>
     questions are waiting for you to start learning the rules of Ultimate!
  </Text>
  <View
    style={
      {
        "marginHorizontal": 16,
      }
    }
  >
    <View
      style={
        [
          {
            "borderRadius": 3,
            "overflow": "hidden",
          },
          {
            "borderRadius": 3,
          },
          {
            "borderRadius": 8,
            "marginBottom": 16,
          },
          false,
        ]
      }
    >
      <View
        accessibilityRole="link"
        accessibilityState={
          {
            "busy": false,
            "checked": undefined,
            "disabled": false,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#2089dc",
              "borderColor": "#2089dc",
              "borderRadius": 3,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 26,
                "paddingVertical": 1,
                "textAlign": "center",
              }
            }
          >
            Play
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        [
          {
            "borderRadius": 3,
            "overflow": "hidden",
          },
          {
            "borderRadius": 3,
          },
          {
            "borderRadius": 8,
            "marginBottom": 16,
          },
          false,
        ]
      }
    >
      <View
        accessibilityRole="link"
        accessibilityState={
          {
            "busy": false,
            "checked": undefined,
            "disabled": false,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#2089dc",
              "borderColor": "#2089dc",
              "borderRadius": 3,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 18,
                "paddingVertical": 1,
                "textAlign": "center",
              }
            }
          >
            History
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        [
          {
            "borderRadius": 3,
            "overflow": "hidden",
          },
          {
            "borderRadius": 3,
          },
          {
            "borderRadius": 8,
            "marginBottom": 16,
          },
          false,
        ]
      }
    >
      <View
        accessibilityRole="link"
        accessibilityState={
          {
            "busy": false,
            "checked": undefined,
            "disabled": false,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#2089dc",
              "borderColor": "#2089dc",
              "borderRadius": 3,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 18,
                "paddingVertical": 1,
                "textAlign": "center",
              }
            }
          >
            Rules
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      {
        "alignItems": "flex-end",
        "marginHorizontal": 8,
      }
    }
  >
    <Text
      accessibilityRole="link"
      href="/about"
      onPress={[Function]}
      style={{}}
    >
      <Text />
    </Text>
  </View>
</View>
`;
